pkg/template/template_test.go

```diff
@@ -116,7 +116,7 @@ func TestParameterGenerators(t *testing.T) {
 	for i, test := range tests {
 		processor := NewProcessor(test.generators)
 		template := api.Template{Parameters: []api.Parameter{test.parameter}}
-		err, _ := processor.GenerateParameterValues(&template)
+		err := processor.GenerateParameterValues(&template)
```

Comment:
I updated the tests to confirm that the right ErrorType was returned.  I.E. Invalid, or NotFound, or Required.

I may be misinterpreting what you meant by "verifying the field path", but I interpreted it as essentially field.NewPath("template").Child("parameters").Index(i).String(), and if so, that didn't seem very interesting to me.

But confirm/correct me as needed.